### PR TITLE
explicit waits

### PR DESCRIPTION
**Pull request by Daniel2894**

### CODE REVIEW

File: src/page/ProductAddedModal.page.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 
 export class ProductAddedModalPage {
   private get proceedToCheckoutButton(): ElementFinder {
     return element(by.css('[style*="display: block;"] .button-container > a'));
   }
 
-  public checkoutProduct(): promise.Promise<void> {
-    return this.proceedToCheckoutButton.click();
+  public async checkoutProduct(): Promise<void> {
+    await browser.wait(ExpectedConditions.elementToBeClickable(this.proceedToCheckoutButton), 5000);
```
Comment (sorregom) [inline src/page/ProductAddedModal.page.ts:9]: is this the only Expected condition in all the test?

**Review by sorregom — CHANGES_REQUESTED** [review]

File: src/page/ProductAddedModal.page.ts (hunk 1 above)
Comment (Daniel2894) [reply src/page/ProductAddedModal.page.ts:9]: is the only one who fail, after do this the test works fine.

Comment (sorregom) [reply src/page/ProductAddedModal.page.ts:9]: OK
LGTM then

**Review by sorregom — APPROVED** [review]
LGTM

**Pull request merged**